prettier.config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "semi": true,
+  "singleQuote": false,
+  "useTabs": false,
+  "tabWidth": 2,
+  "bracketSpacing": true,
+  "arrowParens": "avoid",
+  "trailingComma": "all",
+  "preferSingleLine": false,
+  "memberExpression": {
+    "linePerExpression": true
+  },
+  "printWidth": 140,
```

Comment:
We go wide!

Reply:
I respect that config with hate on single quotes, trailing comas and "avoid" mode :)